app/src/pages/index.astro

```diff
@@ -12,8 +12,8 @@ import { Button } from "../components";
     mathematics courses.
   </div>
   <div>
-    <Button label="comp 1805" />
-    <Button label="comp 2804" />
+    <Button label="COMP 1805" />
+    <Button label="COMP 2804" />
```

Comment:
Instead of buttons caps in the label, we can use a `text-transform: capitalize;`

Reply:
Yes, that's better, I will fix it!